place_tracker/lib/place_tracker_app.dart

```diff
@@ -1,8 +1,9 @@
 import 'package:flutter/material.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:provider/provider.dart';
 
-import 'app_model.dart';
 import 'place.dart';
+
```

Comment:
I think this newline shouldn't be added. We like to group the different styles of imports.

Reply:
newline has been removed in the latest commit, keeping the import style the same.